R/format.R

```diff
@@ -1,18 +1,28 @@
 #' Assign SAS Format
 #'
-#' Assigns a SAS format from a variable level metadata to a given data frame.
+#' Assigns a SAS format from a variable level metadata to a given data frame. If
+#' no format is found for a given variable, it is set as an empty character
+#' vector. This is stored in the format.sas attribute.
 #'
-#' @param .df A data frame of CDISC standard.
-#' @param metadata A data frame containing variable level metadata.
-#' @param domain A character value to subset the `.df`. If `NULL`(default), uses
-#'   `.df` value as a subset condition.
-#' @param verbose The action the function takes when a variable label isn't.
-#'   found. Options are 'stop', 'warn', 'message', and 'none'
-#' @param metacore `r lifecycle::badge("deprecated")` Previously used to pass metadata now renamed with `metadata`
+#' @inheritParams xportr_length
 #'
 #' @return Data frame with `SASformat` attributes for each variable.
-#' @family metadata functions
-#' @seealso [xportr_label()], [xportr_df_label()] and [xportr_length()]
+#'
```

Comment:
Should we not that messaging is currently not available for this function??

Reply:
I think the lack of 'verbose' argument and no messaging section notes that